cmd/style/style.go

```diff
@@ -0,0 +1,113 @@
+package style
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/charmbracelet/lipgloss"
+)
+
+const (
+	// White represents the common white color used throughout the CLI.
+	White = "252"
+
+	// DarkGrey represents the common dark grey color used throughout the CLI.
+	DarkGrey = "238"
+
+	// Grey represents the common grey color used throughout the CLI.
+	Grey = "248"
+
+	// Red represents the common red color used throughout the CLI.
+	Red = "125"
+
+	// Green represents the common green color used throughout the CLI.
+	Green = "82"
+
+	// Yellow represents the common yellow color used throughout the CLI.
+	Yellow = "214"
+
+	// Purple represnts the common purple color used throughout the CLI.
+	Purple = "139"
```

Comment:
If you use the std Ubuntu terminal like I do, then the purple doesn't look that nicely in contrast to the background color:

![Screenshot from 2024-09-05 14-32-35](https://github.com/user-attachments/assets/b2d30250-c4d7-4f88-825b-49b4492a368e)

Reply:
Good point, I should test out the colors in the default ubuntu terminal theme.